src/include/platform/FailSafeContext.h

```diff
@@ -41,14 +41,20 @@ class FailSafeContext
      */
     CHIP_ERROR ArmFailSafe(FabricIndex accessingFabricIndex, System::Clock::Timeout expiryLength);
     CHIP_ERROR DisarmFailSafe();
+    CHIP_ERROR SetAddNocCommandInvoked(FabricIndex nocFabricIndex);
+    CHIP_ERROR SetUpdateNocCommandInvoked(FabricIndex nocFabricIndex);
 
     inline bool IsFailSafeArmed(FabricIndex accessingFabricIndex) const
     {
         return mFailSafeArmed && MatchesFabricIndex(accessingFabricIndex);
     }
 
+    inline bool IsFailSafeBusy() const { return mFailSafeBusy; }
```

Comment:
This needs to be documented.  Perhaps:
```suggestion
    // Returns true if the fail-safe is in a state where commands that require an armed
    // fail-safe can no longer execute, but a new fail-safe can't be armed yet.
    inline bool IsFailSafeBusy() const { return mFailSafeBusy; }
```